actor/v2action/composite/service_broker_summary.go

```diff
@@ -0,0 +1,138 @@
+package composite
+
+import (
+	"code.cloudfoundry.org/cli/actor/v2action"
+)
+
+//go:generate counterfeiter . ServiceActor
```

Comment:
Please add an extra newline after `//go:generate` so that it doesn't show up in GoDoc.

Reply:
Wow that was quick :)

Reply:
These are not the droids you're looking for